src/post_section/post_section.jsx

```diff
@@ -1,131 +1,172 @@
+                            <tr className="table_of_contact">
+                                {links.map((link) => (
+                                    <LinkList link={link} />
+                                ))}
+                            </tr>
+                        </table>
                     </div>
-                    <div className="rectangle">
-                        <picture>
-                            <source media="(max-width:854px)" width="375" height="250" srcset="/photos/rectangle.png" />
-                            <source media="(min-width:854px)" width="854" height="570" srcset="/photos/rectangle_desktop.jpg" />
-                            <img src="/photos/rectangle_desktop.jpg" width="854" height="570" alt="rectangle" />
-                        </picture>
-                        <figcaption className="rectangle_caption">Image caption centered this way and I’ll make this a bit longer to
-                            indicate the amount of line-height</figcaption>
+                </div>
+                <div className="rectangle">
+                    <picture>
+                        <source
+                            media="(max-width:854px)"
+                            width="375"
+                            height="250"
+                            srcset={data.posts.thumbnail_url}
+                        />
+                        <source
+                            media="(min-width:854px)"
+                            width="854"
+                            height="570"
+                            srcset={data.posts.thumbnail_url}
+                        />
+                        <img
+                            src={data.posts.thumbnail_url}
+                            width="854"
+                            height="570"
+                            alt="main photo"
+                        />
+                    </picture>
+                    <figcaption className="rectangle_caption">
+                        {data.posts.title}
+                    </figcaption>
+                </div>
+                <div className="second_section_block">
+                    {
+                        <div
+                            className="post_body"
+                            dangerouslySetInnerHTML={{ __html: data.posts.content }}
+                        />
+                    }
+                    <div className="article_section_footer">
+                        <p className="section_footer_text share">Share:</p>
+                        <table className="table_footer">
+                            <tr className="table_of_contact_second">
+                                {links.map((link) => (
+                                    <LinkListBottom link={link} />
+                                ))}
+                            </tr>
+                        </table>
+                        <p className="section_footer_text tags">
+                            Tags:{" "}
```

Comment:
why do you render an empty string here? you have a non-breaking space if you need one
```suggestion
                            Tags:&nbsp;
```